### PR TITLE
Bump httpntlm version to fix the non dev high audit error

### PR DESCRIPTION
**Pull request by Nib-Cliffy**
Httpntlm version bump to fix the underscore security vulnerability 

https://www.npmjs.com/advisories/1674

### CODE REVIEW

**Comment by smokhov** [comment]
Folks, when are you planning on merging and publishing this?

**Comment by chris-williams-efr** [comment]
I had a look on the Gitter for this package, where this PR was mentioned, but couldn't see any responses from maintainers about merging this PR.  What's the best form of comms, as issues are disabled?

**Comment by jsdevel** [comment]
please pull latest

**Comment by jbrizio** [comment]
Please @Nib-Cliffy update your fork and resolve conflicts to allow merge this.

**Comment by Nib-Cliffy** [comment]
Merged master resolved conflicts, also changed version to 0.39.0

**Comment by jbrizio** [comment]
Hi @jsdevel, this pull request is ready to merge! Thanks

**Comment by Nib-Cliffy** [comment]
Hi @jsdevel , is there anything holding this up from merging to master ?

File: package.json
Comment (jsdevel) [inline package.json:3]: please revert this version bump.  we'll take care of that when we publish.

**Review by jsdevel — CHANGES_REQUESTED** [review]

File: package.json
Comment (Nib-Cliffy) [reply package.json]: done

**Comment by SanketDorle** [comment]
I am also waiting for the upgraded version of httpntlm, please lets us know by when it will get update .

File: package.json
Comment (Nib-Cliffy) [reply package.json]: Please tell me this isn't going to  take another 2 weeks to approve :)

Comment (smokhov) [reply package.json]: Sigh, looks like have to wait till at least 0.40.0

**Comment by jsdevel** [comment]
please fix the conflicts

**Comment by smokhov** [comment]
This is a long standing vulnerability fix, why was this not merged first and foremost before other PRs? This is more pressing especially for the dependent projects waiting on this.

**Comment by smokhov** [comment]
Is it possible #1144 obsoletes this PR since it removes httpntlm if I understand correctly? Though it seems #1144 broke something... @jsdevel @Nib-Cliffy

**Comment by RopoMen** [comment]
Hi,
There is now another security issue (https://www.npmjs.com/advisories/1674) with ``ùnderscore``` that is related to old ```httpntlm``` package. This new issue should be fixed by upgrading to the latest httpntml version that is 1.7.7

and then criticism, is this the way how ```node-soap``` handles security related issues?

Br,
Markku

**Comment by jsdevel** [comment]
@Nib-Cliffy  please update with the latest from master.  i had to migrate to github actions.

**Comment by smokhov** [comment]
@jsdevel -- this PR can be closed since you merged its successor in #1158

**Pull request closed without merging**
